scripts/deploy-production.sh

```
@@ -0,0 +1,351 @@
+      echo "Unknown option: $1"
+      exit 1
+      ;;
+  esac
+done
+
+# Logging function
+log() {
+  local level=$1
+  shift
+  local message="$*"
+  local timestamp=$(date '+%Y-%m-%d %H:%M:%S')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)